pkg/admin/src/db.ts

```diff
@@ -8,25 +8,25 @@ import {
   JoinColumn,
```

Comment:
These changes should really be separate but I think fine to mix this in. Was mostly just a model change so i could update a user's name for them.